core/src/main/resources/reference.conf

```diff
@@ -265,6 +265,9 @@ call-caching {
   # to fail for external reasons which should not invalidate the cache (e.g. auth differences between users):
   # (default: true)
   invalidate-bad-cache-results = true
+
+  # The maximum number of times Cromwell will attempt to copy cache hits before giving up and running the job.
+  max-failed-copy-attempts = 1000000
```

Comment:
oh interesting I suppose we could do either or both of those

Reply:
Happy to stick with a "max attempts" counter for now and consider a timer in the future if it's not sufficient

Reply:
[TOL] I feel like the end user goal of not making people get stuck call caching is better served by a time limit.

If there are a few users competing for fixed GCS bandwidth, it could take an unpredictable, possibly long, amount of time to reach their attempts.

Reply:
This decision seems producty so I would defer to our esteemed PO. If we do decide to make this a time limit that could be added to the "polish up attempt limiting after the release" ticket that needs to be made.